typescript-conditionals/main.ts

```diff
@@ -0,0 +1,101 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(number: number): any {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number: number): any {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string: string): any {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person: { name: string; age: number }): any {
+  if (person.age < 21) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function isOldEnoughToDrive(person: { name: string; age: number }): any {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person: {
+  name: string;
+  age: number;
+}): any {
+  if (person.age > 0) {
```

Comment:
Condition is incorrect. They want you to use AND operator to combine 2 conditions (drink and drive)